src/benchmark_mediation.py

```diff
@@ -66,20 +66,22 @@ def get_interactions(interaction, *args):
            [ 2.,  3.,  1.,  2.,  2.,  3.,  4.,  6.,  2.],
            [ 4.,  5.,  1.,  2.,  4.,  5.,  8., 10.,  2.]])
     """
-    variables = args
+    variables = list(args)
```

Comment:
is there any reason to sue this patterns. Arguments should be passed explicitly.

Reply:
actually you can specify a list of variables between which you want to compute interaction terms. Do you think this should be done differently?